sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_model.py

```diff
@@ -70,6 +70,7 @@ def calibrate(
             image_points.astype(np.float32).reshape(-1, 1, 2) for image_points in image_points_list
         ]
 
+        # cSpell:ignore rvecs, tvecs
```

Comment:
remove here?

Reply:
Done !
(the rvec, tvec entry did not apply here so I had to add them too)